src/ucp/core/ucp_ep.c

```diff
@@ -458,6 +456,7 @@ ucs_status_t ucp_worker_create_mem_type_endpoints(ucp_worker_h worker)
             goto err_free_address_list;
         }
 
+        ucp_ep_flush_state_reset(worker->mem_type_ep[mem_type]);
```

Comment:
is it needed for memtype ep?

Reply:
but i thought we connected memtype ep to worker addr?

Reply:
> but i thought we connected memtype ep to worker addr?

yes, we do it. but it doesn't reset flush_state